lib/clients/BaseClient.py

```diff
@@ -70,6 +71,35 @@ def __init__(self, operation_name, configuration, directory_persistent, director
         self.__mounted_devices = []
         self.__devices = {}
 
+    def _get_credentials_from_credhub(self, *args):
+        access_token = self.__retry(self.__getAccessToken, args)
+        params = [args[0], access_token]
+        return self.__retry(self.__get_credentials_from_credhub, params)
+    
+    def __get_credentials_from_credhub(self, configuration, access_token):
+        access_token = self.__getAccessToken(configuration)
```

Comment:
Why we need to again get token here? Already called https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/35/files#diff-973b6bc10f6b36c84d0dbeb7f11dc11aR75